frontend/src/locales/en.json

```
@@ -535,7 +541,7 @@
     "googlePermissionEventsName": "View and edit events on all your calendars",
     "homepage": {
       "calendarCopy": "Connect your calendar and see all your meetings in one place. Book one-off or recurring events and invite people to join you.",
-      "copywrite": "Thunderbird is now part of {mzlaLink}, a wholly owned subsidiary of Mozilla Foundation. Portions of this content are ©1998–2024 by individual contributors. Content available under a {creativeCommonsLink}.",
+      "copywrite": "Thunderbird is now part of {mzlaLink}, a wholly owned subsidiary of Mozilla Foundation. Portions of this content are ©1998–2025 by individual contributors. Content available under a {creativeCommonsLink}.",
```

```suggestion
      "copywrite": "Thunderbird is now part of {mzlaLink}, a wholly owned subsidiary of Mozilla Foundation. Portions of this content are ©1998–{currentYear} by individual contributors. Content available under a {creativeCommonsLink}.",
```

Then when calling it use the magic of js to set the year? :smile: